src/zgamma/plot_peakinghiggsbkgs.cxx

```diff
@@ -0,0 +1,151 @@
+                      {spath_2016+"*HToGG*.root", spath_2016APV+"*HToGG*.root", spath_2017+"*HToGG*.root", spath_2018+"*HToGG*.root"},     "1");
+
+  auto proc_htt     = Process::MakeShared<Baby_pico>("H #rightarrow #tau#tau",     back, TColor::GetColor("#ff9632"),
+                      {spath_2016+"*HToTauTau*.root", spath_2016APV+"*HToTauTau*.root", spath_2017+"*HToTauTau*.root", spath_2018+"*HToTauTau*.root"}, "1");
+
+  auto proc_hWW     = Process::MakeShared<Baby_pico>("H #rightarrow WW",           back, TColor::GetColor("#1a76a9"), 
+                      {spath_2016+"*HToWW*.root", spath_2016APV+"*HToWW*.root", spath_2017+"*HToWW*.root", spath_2018+"*HToWW*.root"},     "1");
+
+  auto proc_hZZ     = Process::MakeShared<Baby_pico>("H #rightarrow ZZ",           back, TColor::GetColor("#08bab7"), 
+                      {spath_2016+"*HToZZ*.root", spath_2016APV+"*HToZZ*.root", spath_2017+"*HToZZ*.root", spath_2018+"*HToZZ*.root"},     "1");
+
+  //vector containing the procs
+  vector<shared_ptr<Process>> procs = {proc_hmumu,proc_hgg,proc_htt,proc_hWW,proc_hZZ};
+
+
+  vector<NamedFunc> lep  =  {"1","ll_lepid[0]==11","ll_lepid[0]==13"};
+  vector<string>    slep =  {"ll","el","mu"};
+
+  //Baseline selection for analysis
+  vector<NamedFunc> cutflow = {"1", pass_trigs_and_pt, "nll>0", "nllphoton > 0","ll_m[0] > 50", "photon_pt[0] > 15", "photon_pt[0]/llphoton_m[0] > 15.0/110.0", "llphoton_m[0] > 100 && llphoton_m[0] < 180", 
+                               "ll_m[0] + llphoton_m[0] > 185", "photon_id80[0]" ,"ll_m[0] > 80 && ll_m[0] < 100" };
+  vector<string> cutflow_lab = {"All events", "Triggers and $p_{T}(l)$ selections", "$N_{ll} > 0$", "$m_{ll} >$ 50 GeV" ,"$N_{ll\\gamma} > 0$", "$p_{T}(\\gamma) > 15$ GeV", 
+                                "$p_{T}(\\gamma)/m_{ll\\gamma} > 15/110$", "100 GeV $< m_{ll\\gamma} < 180$ GeV", "$m_{ll} + m_{ll\\gamma} > 185$ GeV", "\\gamma IDMVA WP80" ,"80 GeV $< m_{ll} < 100$ GeV "};
+
+  for(unsigned int idx_i = 2; idx_i < cutflow.size(); idx_i++){
+    cutflow[idx_i] = cutflow[idx_i - 1] && cutflow[idx_i];
+  }
+  NamedFunc r2_baseline    = r2_baseline;
+  NamedFunc tight_baseline = tightened_baseline;
+
+
+  //Plot options used in this code
+  PlotOpt lin_stack("txt/plot_styles.txt","CMSPaper");
+  lin_stack.Title(TitleType::info).Stack(StackType::signal_overlay)
+           .UseCMYK(false) .FileExtensions({"pdf"});
+  vector<PlotOpt> ops = {lin_stack};
+  vector<PlotOpt> ops_ls = {lin_stack().Stack(StackType::lumi_shapes)};
+  vector<PlotOpt> logops = {lin_stack().YAxis(YAxisType::log)};
+ 
+  //Start making plots
+  PlotMaker pm;
+
+  //Loop containing which plots are made
+  for(unsigned int idx_i(0); idx_i < lep.size(); idx_i++) { 
+    pm.Push<Hist1D>(Axis(80,100,180, "llphoton_m[0]",    "m_{ll#gamma} [GeV]", {122,128}), r2_baseline && lep.at(idx_i), procs, ops).Weight(wgt_1invfb).Tag("ShortName:hpb_");
```

Comment:
I am having a difficult time understanding what wgt_1invfb is. Where is it defined?